### DISCUSSION TITLE
Storage - Internal Server Error - Cannot read property 'fields' of undefined

### DISCUSSION THREAD

**Discussion by PrestoAnomaly** [supabase/supabase#2112] [Questions]
I'm having an issue with storage. The upload completes to the bucket successfully, but the object returned is:

```
Object {
  "data": null,
  "error": Object {
    "error": "Internal Server Error",
    "message": "Cannot read property 'fields' of undefined",
    "statusCode": 500,
  },
}
```

Previously, a URL to the uploaded image would be returned.  Any help or suggestions would be greatly appreciated!

**Comment by inian** [comment]
Can you send me the API URL here or to beta@supabase.io @PrestoAnomaly? Thanks

**Reply by PrestoAnomaly** [reply]
@inian Can you confirm you received an email? Want to make sure my mail server is still working properly!

**Reply by inian** [reply]
Yup, got it! Replied to your email @PrestoAnomaly

**Comment by PrestoAnomaly** [comment]
@inian I sent it via email. Thanks!

**Reply by danielolamide** [reply]
Hey @PrestoAnomaly , were you able to get to solve this?
